js/web/docs/webnn-operators.md

```diff
@@ -0,0 +1,96 @@
+| AveragePool | ai.onnx(7-9, 10, 11, 12-18, 19+) | averagePool2d | ✓ | ✓ | Only supports 4-D input, 2-D 'kernel_shape', 'count_include_pad' value is 0 |
+| BatchNormalization | ai.onnx(7-8, 9-13, 14, 15+) | batchNormalization | ✗ | ✓ | Only supports 'training_mode' value is 0, one output |
+| Cast | ai.onnx(7-8, 9-12, 13-18, 19-20, 21+) | cast | ✗ | ✓ | |
+| Ceil | ai.onnx(7-12, 13+) | ceil | ✓ | ✓ | |
+| Clip | ai.onnx(7-10, 11, 12, 13+) | clamp | ✓ | ✓ | |
+| Concat | ai.onnx(7-10, 11-12, 13+) | concat | ✓ | ✓ | |
+| Conv | ai.onnx(7-10, 11+) | conv2d | ✓ | ✓ | Only supports 3-D or 4-D input and 'W' (weight). WebNN CPU requires the 'W' (weight) input to be a constant |
+| ConvInteger | ai.onnx(10+) | conv2dInteger | ✗ | ✗ | Only supports 3-D or 4-D input and 'W' (weight) |
+| ConvTranspose | ai.onnx(7-10, 11+) | convTranspose2d | ✓ | ✓ | Only supports 3-D or 4-D input and 'W' (weight). WebNN CPU requires the 'W' (weight) input to be a constant |
+| Cos | ai.onnx(7+) | cos | ✗ | ✓ | |
+| Div | ai.onnx(7-12, 13, 14+) | div | ✓ | ✓ | |
+| DequantizeLinear | ai.onnx(10-12, 13-18, 19-20, 21+) | dequantizeLinear | ✗ | ✗ | |
+| DynamicQuantizeLinear | ai.onnx(11+) | dynamicQuantizeLinear | ✗ | ✗ | |
+| Elu | ai.onnx(7+) | elu | ✓ | ✓ | |
+| Equal | ai.onnx(7-10, 11-12, 13-18, 19+) | equal | ✗ | ✓ | |
+| Erf | ai.onnx(7-9, 10-12, 13+) | erf | ✗ | ✓ | |
+| Exp | ai.onnx(7-12, 13+) | exp | ✗ | ✓ | |
+| Expand | ai.onnx(8-12, 13+) | expand | ✗ | ✓ | 'shape' input should be a constant |
+| Flatten | ai.onnx(7-8, 9-10, 11-12, 13-20, 21+) | reshape | ✓ | ✓ | |
+| Floor | ai.onnx(7-12, 13+) | floor | ✓ | ✓ | |
+| Gather | ai.onnx(7-10, 11-12, 13+) | gather | ✗ | ✓ | |
+| Gelu | ai.onnx(20+) | gelu | ✗ | ✓ | |
+| Gemm | ai.onnx(7-8, 9-10, 11-12, 13+) | gemm | ✓ | ✓ | Only supports 1-D 'C' input |
+| GlobalAveragePool | ai.onnx(7+) | averagePool2d | ✓ | ✓ | Only supports 4-D input |
+| GlobalMaxPool | ai.onnx(7+) | maxPool2d | ✓ | ✓ | Only supports 4-D input |
+| GlobalLpPool| ai.onnx(7+) | l2Pool2d | ✗ | ✓ | Only supports 4-D input, 'p' value is 2 |
+| Greater | ai.onnx(7-8, 9-12, 13+) | greater | ✗ | ✓ | |
+| GreaterOrEqual | ai.onnx(12-15, 16+) | greaterOrEqual | ✗ | ✓ | |
+| HardSigmoid | ai.onnx(7+) | hardSigmoid | ✗ | ✓ | |
+| HardSwish | ai.onnx(14+) | hardSwish | ✓ | ✓ | |
+| Identity | ai.onnx(7-13, 14-15, 16-18, 19-20, 21+) | identity | ✗ | ✓ | |
+| InstanceNormalization | ai.onnx(7+) | instanceNormalization | ✗ | ✓ | |
+| LayerNormalization | ai.onnx(7-16, 17+) | layerNormalization | ✗ | ✓ | |
+| LeakyRelu | ai.onnx(7-15, 16+) | leakyRelu | ✓ | ✓ | |
+| Less | ai.onnx(7-8, 9-12, 13+) | lesser | ✗ | ✓ | |
+| LessOrEqual | ai.onnx(12-15, 16+) | lesserOrEqual | ✗ | ✓ | |
+| Log | ai.onnx(7-12, 13+) | log | ✗ | ✓ | |
+| LpPool | ai.onnx(7-10, 11-17, 18+) | l2Pool2d | ✗ | ✓ | Only supports 4-D input, 2-D 'kernel_shape', 'p' value is 2 |
+| MatMul | ai.onnx(7-8, 9-12, 13+) | matmul | ✓ | ✓ | WebNN CPU doesn't support broadcasting for MatMul |
```

Comment:
Besides, WebNN CPU only supports input rank > 3.

Reply:
Same here, we did workaround, use `gemm` for 2D case, and reshape to handle 1D case.

Reply:
Nice, ack!